<!--
  ~ Copyright (c) Fundacion Jala. All rights reserved.
  ~ Licensed under the MIT license. See LICENSE file in the project root for full license information.
  -->

<h2>Noncompliant Code Example</h2>
<p>The maximum number of statements in method is 50</p>


<h2>Compliant Solution</h2>

<pre>

</pre>
